usrsctplib/netinet/sctp_pcb.c

```diff
@@ -3004,7 +3004,7 @@ sctp_inpcb_alloc(struct socket *so, uint32_t vrf_id)
 
 	/* Setup the initial secret */
 	(void)SCTP_GETTIME_TIMEVAL(&time);
-	m->time_of_secret_change = time.tv_sec;
+	m->time_of_secret_change = (unsigned int)time.tv_sec;
```

Comment:
Fixed in https://github.com/sctplab/usrsctp/commit/f2c2b601285fec3289c483088ffef255598b585f.